tetris.js

```diff
@@ -309,5 +309,14 @@ module.exports = {
 		let coordinateDifference_Y = coordinatesPivot.x - coordinatesElement.x
 		return { x: coordinatesPivot.x - coordinateDifference_X, y: coordinatesPivot.y + coordinateDifference_Y }
 	},
-	rotatesArrayOfCoordinates: function(){}
+	rotatesArrayOfCoordinates: function(arrayCoordinates, coordinatePivot){
+		let localArrayCoordinates = []
+		if(arrayCoordinates.length === 0){
```

Comment:
Is this check necessary?

Reply:
I think yes
If array === 0  then it is empty

Reply:
Ok, why is in necessary?

Reply:
This check is not needed, remove

Reply:
Done